### PR TITLE
Added MUI icons to database and a Javascript course in Spanish

### PR DESCRIPTION
**Pull request by franciscoluna-28**
After a long time, I finally implemented these changes correctly. Here are the things I added to the project:

image
image

Hopefully the commit will work well.

### CODE REVIEW

**Comment by franciscoluna-28** [comment]
Also, my apologies for misunderstanding git add before... I was trying to merge ALL the changes at once because of a mistake that I did before and implemented those changes. This time, I decided to start from scratch and read the documentation slowly since I'm new to contributing to open source. Anyway, fortunately the changes I've made will help the project a bit :)

File: database/languages/javascript.json
Comment (rupali-codes) [inline database/languages/javascript.json:1]: 'category' should be **languages** not **other**

**Review by franciscoluna-28 — COMMENTED** [review]
Updated "Others" to "Languages"

File: database/languages/javascript.json
Comment (franciscoluna-28) [reply database/languages/javascript.json:1]: Fixed the issue

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @franciscoluna-28! :) There are some conflicts that need to be solved.

**Review by rupali-codes — APPROVED** [review]
lgtm, thanks

**Comment by rupali-codes** [comment]
@franciscoluna-28 could you please resolve the conflicts so we can merge this PR :)

**Review by franciscoluna-28 — COMMENTED** [review]
Added the new course and fixed the existent conflict

**Comment by rupali-codes** [comment]
@franciscoluna-28 Still have conflicts

**Comment by rupali-codes** [comment]
@franciscoluna-28 are you still working on this?

**Comment by CBID2** [comment]
Hey @franciscoluna-28! :) Are you still working on this PR?

**Comment by franciscoluna-28** [comment]
I'm taking a break on Open source. My apologies for the late response. I need to learn more about Github before contributing again.

**Pull request closed without merging**